container/container_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 	containertypes "github.com/docker/docker/api/types/container"
 )
 
+const (
+	containerSecretMountPath = `C:\ProgramData\Docker\secrets`
```

Comment:
This confused me too initially. It's a destination path inside the container which will always be the on the C: drive. I think it should probably renamed as `secretMountPathInsideContainer` or similar to avoid confusion.